autotest/autotest.cpp

```diff
@@ -2006,6 +2016,42 @@ bool tAutotest::step_dumpPackets(const YAML::Node& yamlStep,
 	return true;
 }
 
+bool tAutotest::step_bird(const YAML::Node& yamlStep,
+                          const std::string& path)
+{
+	std::string confPath;
+	for (const auto& yamlIter : yamlStep)
+	{
+		confPath = yamlIter.as<std::string>();
+	}
+
+	if (!confPath.empty())
+	{
+		const std::string execCommand = "bird -c " + path + confPath;
+
+		std::string resExec = exec(execCommand.c_str());
+
+		printf("%s", resExec.data());
```

Comment:
just
```suggestion
		std::cout <<resExec;
```

Or use `YANET_LOG_`_something_. Using plain `printf` seems strange, after all, we're using C++